src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPerDay = 40;
+
+  if (days < 3) {
+    return days * rentPerDay;
+  }
+
+  if (days < 7) {
+    return days * rentPerDay - 20;
+  }
+
+  if (days >= 7) {
+    return days * rentPerDay - 50;
+  }
```

Comment:
maybe would be better to put formula (days * 40) in one variable to write lesser code? and u will delete one if